File: python_learn/day01/de01.call.function.py
# -*- coding: utf8 -*-
def result():
    print('111')
    return [1, 2]

# result在此处会调用与js的调用不同


def test(s=result()):
    s.append('end')
    print(s)


print(__name__)
test()
test()
test()

File: python_learn/day01/de01.call.function.js
function test(s = result()) {
	s.push("end");
	console.log(s);
}
function result() {
	console.log("1111111");
	return [1, 2];
}

test([1, 27]);
test([1, 27]);
test([1, 27]);

try {
	throw 11111;
} catch (error) {
	console.log(error);
}
console.log(2);
